.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: [2.4.5, 2.5.3]
+        puppet-gem-version: ["~> 5.0", "~> 6.0"]
+        rubygems-version: [3.1.0]
+    env:
+      PUPPET_GEM_VERSION: ${{ matrix.puppet-gem-version }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+    - name: Update Rubygems
+      run: gem update --system ${{ matrix.rubygems-version }}
+    - name: Bundle Install
+      run: bundle install
+    - name: Check symlinks
+      run: bundle exec rake check:symlinks
```

Comment:
Hmm, from looking at the `travis.yml` file, I think all the non-unit-test-rake-tasks ran only once? Here they seem to run for both PUPPET_GEM_VERSIONS. Is that intentional?

Reply:
Also don't see the unit tests (`parallel_spec`) here.

Reply:
I'll make the static tests run only once, but there currently aren't any valid unit or acceptance tests.

Reply:
No worries, I was basing this review off "is everything identical to how it was done in `.travis.yml`" so if you've found that some stuff didn't make sense/isn't needed, that's cool too.

Reply:
That's a good point though. There was an attempt at acceptance tests that don't currently work. I'll file a ticket to fix them and add them here.

Reply:
removed them for now. I'll get the acceptance tests back up and running in another PR since they currently aren't functional anyways.

Reply:
Sounds good. Only other thing I recommend is to check if there's a way for GA to print the invoked command line-by-line? That way, I can just see `bundle exec rake check:git_ignore` and know that's the `.gitignore` step. As a possible maintainer, it is annoying that I have to provide a `name:` key anytime I want to add a new step to CI.

Reply:
I ended up mirroring the way we have it set up in the cmdb_integration. All of the checks are bundled together so you don't have to give them individual names but it still seems easy to see which one failed.

Reply:
Cool, thanks. Can you do something similar w/ the Release job re: eliminating the `name` key? It's ok if what you do just prints the command, that's more than enough.

Reply:
I combined the last couple of commands under a single run key so it will be easy to add and/or modify steps. The previous steps utilize other github actions which seem to require a name key. These probably won't be changed often anyways.

Reply:
Ah ok, makes sense. Thanks.